app_dart/bin/server.dart

```diff
@@ -45,6 +45,12 @@ Future<void> main() async {
       config,
     );
 
+    /// Cocoon scheduler service to manage validating commits in presubmit and postsubmit.
+    final Scheduler scheduler = Scheduler(
+      cache: cache,
```

Comment:
Could we implement cache in a separate PR when needed? It is not clear to me how it helps in this PR. It will help if any issue pops up related to cache (like a revert without affecting other implementations).

Reply:
Removed. This was left over as this PR is a split from a larger PR.